client/src/components/new-request/quick-search-results.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <MainContainer id="quick-search-results-container">
+    <template v-slot:container-header>
+      <v-col cols="auto" class="h6 py-0 mt-1">
+        You are searching for a name for a
+        {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }}
+        {{ entityText }}
+      </v-col>
+    </template>
+    <template v-slot:content>
+      <v-row no-gutters class="pt-3">
+        <NameInput/>
+      </v-row>
+      <v-row justify="center" class="ma-0 pa-0">
```

Comment:
FYI, "no-gutters" is basically equivalent to "ma-0 pa-0" for the row as well as the columns in it. It's not always the best thing to do but FYI.

Reply:
I was wondering what that was haha